src/sklearn_evaluation/plot/grid_search.py

```diff
@@ -139,7 +139,7 @@ def _grid_search_single(grid_scores, change, subset, kind, ax):
         # take (param, value) and convert them to 'param: value'
         label = ['{}: {}'.format(*t) for t in params_kv]
         # now convert it to one string
-        label = reduce(lambda x, y: x + ', ' + y, label, '')
+        label = ', '.join(label)
```

Comment:
this is breaking the tests, it'll produce a string that starts with `,` - please revert the change
![single_numeric_partially_restricted](https://user-images.githubusercontent.com/989250/167231878-c2e67349-e27c-45e7-8cd2-764e11f97993.png)

Reply:
Ok I will add the `sort` argument.
However my change line 142 for the labeling actually fixes the leading comma problem.

Reply:
```py
from functools import reduce

label = ['criterion: entropy', 'max_features: sqrt']
label_bad = reduce(lambda x, y: x + ', ' + y, label, '')
label_good = ', '.join(label)

print('label_bad =', label_bad)
print('label_good =', label_good)
```
will output
```
label_bad = , criterion: entropy, max_features: sqrt
label_good = criterion: entropy, max_features: sqrt
```

Reply:
I updated the baseline images for the tests